### ISSUE TITLE
WebHooks support

### ISSUE THREAD

**Issue by giuliov** [tfsaggregator/tfsaggregator#138]
Support VSTS/TFS webhooks.
### Advantages
- Aggregator can be used with VSTS.
- Remove need to install plugin on AT
### Issues
- Manage credentials
### Design

Add a web project to get the calls
Receiver creates event object similar to server plugin event
Extend Setup to add this DLL (option to create Web in IIS)

**Comment by giuliov** [comment]
Impacted by #134

**Comment by jessehouwing** [comment]
I think we should put this in a separate installer. I love the idea! For VSTS, do we expect people to install this as a webrole/azure website? Or is the expectation that we can host TFS Aggregator as a service?

**Comment by giuliov** [comment]
We can stay with one MSI: at some point I added the selection of features to install.

I think there are three scenarios:

1. On-premises for testing and decoupling from production AT
2. Private VSTS support, where people have anything from a VM to a webapp
3. TFS Aggregator as-a-service

but we may need a sponsor or start a business for scenario 3 :-)

BTW I pushed a PoC in the `feature/webhooks` branch a while ago.
